fili-sql/src/main/java/com/yahoo/bard/webservice/sql/DruidQueryToSqlConverter.java

```diff
@@ -151,13 +152,14 @@ public String buildSqlQuery(DruidAggregationQuery<?> druidQuery, ApiToFieldMappe
      *
      * @return the sql equivalent of the query.
      */
-    private RelNode convertDruidQueryToRelNode(
+    protected RelNode convertDruidQueryToRelNode(
             DruidAggregationQuery<?> druidQuery,
             ApiToFieldMapper apiToFieldMapper,
             SqlPhysicalTable sqlTable
     ) {
-        RelBuilder builder = calciteHelper.getNewRelBuilder(sqlTable.getSchemaName());
-        return builder.scan(sqlTable.getName())
+        RelBuilder builder = calciteHelper.getNewRelBuilder(sqlTable.getSchemaName(), null);
```

Comment:
Why are we passing a hardcoded `null` here instead of the `sqlTable.getCatalog()`?

Reply:
For original Sql implementation catalog is always null.